### PR TITLE
Viber support

### PR DESCRIPTION
**Pull request by rodrigokamada**
Dear,

I am adding support for Viber in this PR commented in this issue [https://github.com/MurhafSousli/ngx-sharebuttons/issues/380](https://github.com/MurhafSousli/ngx-sharebuttons/issues/380).

Thanks!

### CODE REVIEW

**Comment by MurhafSousli** [comment]
Hi @rodrigokamada, thanks for the PR, did you test it out using the custom share button?

File: README.md [hunk 1]
```diff
@@ -29,21 +29,22 @@ For the documentation see the [WIKI](https://github.com/MurhafSousli/ngx-sharebu
...
-8. Tumblr
-9. Telegram
-10. Email
-11. SMS
-12. VK
-13. Mix
-14. Xing
-15. Copy link
-16. Print
+1. Twitter
```
Comment (MurhafSousli) [inline README.md:32]: Can you fix the numbers?

**Review by MurhafSousli — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rodrigokamada) [reply README.md:32]: I changed it to use the ordered list like this: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists

**Comment by rodrigokamada** [comment]
> Hi @rodrigokamada, thanks for the PR, did you test it out using the custom share button?

Hi, I had already tested it on my machine using the `ngx-sharebuttons-demo` project.

And now I tested it using the custom button with the code below.

```
import { faViber } from '@fortawesome/free-brands-svg-icons/faViber';

this.share.addButton('customButton', {
  type: 'customButton',
  text: 'customButton',
  color: '#665ca7',
  icon: faViber,
  func: () => of({}).pipe(tap(() => window.open('viber://forward?text=Share content', '_blank')))
});
```

**Comment by rodrigokamada** [comment]
Hi @MurhafSousli ,

This:
1. Test 1
1. Test 2
1. Test 3

is the same effect the this:
1. Test 1
2. Test 2
3. Test 3

in the README file.

So, in the second case, if you add the item in the middle, you will need to change the others itens. And in the first, you do not need to change.

**Review by MurhafSousli — APPROVED** [review]

**Pull request merged**